lms/migrations/versions/e69454a18f38_add_file_table.py

```diff
@@ -0,0 +1,50 @@
+"""
+Add file table.
+
+Revision ID: e69454a18f38
+Revises: bb11b5b06274
+Create Date: 2021-06-15 12:08:04.564355
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "e69454a18f38"
+down_revision = "bb11b5b06274"
+
+
+def upgrade():
+    op.create_table(
+        "file",
+        sa.Column(
+            "created",
+            sa.DateTime(),
+            server_default=sa.func.now(),
+            nullable=False,
+        ),
+        sa.Column(
+            "updated",
+            sa.DateTime(),
+            server_default=sa.func.now(),
+            onupdate=sa.func.now(),
+            nullable=False,
+        ),
+        sa.Column("id", sa.Integer(), autoincrement=True, primary_key=True),
+        sa.Column(
+            "application_instance_id",
+            sa.Integer(),
+            sa.ForeignKey("application_instances.id", ondelete="cascade"),
+            nullable=False,
+        ),
+        sa.Column("type", sa.String(), nullable=False),
+        sa.Column("lms_id", sa.String(), nullable=False),
+        sa.Column("course_id", sa.String()),
+        sa.Column("name", sa.String()),
```

Comment:
We've been using `sa.Unicode()` or `sa.UnicodeText()`expliclty in other PRs

Reply:
Does this require the model to be updated to match?

Reply:
Yes, the column has to have the same `sa.Unicode()` type on the model.